tests/Drupal/Tests/Driver/Kernel/FieldHandlerKernelTestBase.php

```
@@ -0,0 +1,166 @@
+      ->loadUnchanged($stub->id);
+    $this->assertInstanceOf(ContentEntityInterface::class, $reloaded);
+
+    // Some handlers (e.g. ImageHandler) emit a flat associative array as
+    // single-delta shorthand rather than a list of deltas. Normalise that
+    // shape into a one-element list so the iteration below is uniform.
+    $deltas = is_array($stub->$field_name) && !array_is_list($stub->$field_name)
+      ? [$stub->$field_name]
+      : $stub->$field_name;
+
+    foreach ($deltas as $delta => $expected) {
+      $item = $reloaded->get($field_name)->get($delta);
```

PHPStan (2.1.46)

[error] Call to an undefined method Drupal\Core\Entity\EntityInterface::get().


(method.notFound)